package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "scripts": {
     "prestart": "if [ ! -f ./variable-options.yaml ]; then touch ./variable-options.yaml; fi",
-    "start": "webpack-dev-server --host 0.0.0.0 --history-api-fallback",
+    "start": ". ./generate-commitish.sh; webpack-dev-server --host 0.0.0.0 --history-api-fallback",
```

Comment:
Rather than having the script set an environment variable (side effects == bad), use the output of the script to set the variable on the run line. E.g.:

```
CE_CURRENT_VERSION=$(./generate-commitish.sh) webpack-dev-server --host 0.0.0.0 --history-api-fallback
```